accelerator/core/serializer/serializer.c

```diff
@@ -517,6 +517,13 @@ status_t ta_send_transfer_req_deserialize(const char* const obj, ta_send_transfe
       req->msg_len = msg_len * 3;
       flex_trits_from_trytes(req->message, req->msg_len, (const tryte_t*)json_result->valuestring, msg_len, msg_len);
     }
+
+    if (req->msg_len > NUM_TRYTES_MESSAGE) {
+      ret = SC_SERIALIZER_MESSAGE_LEGNTH;
+      ta_log_error("%s\n", "SC_SERIALIZER_MESSAGE_LEGNTH");
```

Comment:
Improve the error message. Add `exceed` or `out of range` into the description.
Integrate it into `ta_error_to_string`